src/server/transport/socketio.js

```diff
@@ -73,7 +74,7 @@ export function SocketIO({
       });
 
       app.context.io = io;
-      io.attach(app);
+      io.attach(app, https != null, https);
```

Comment:
Could you use `!!https` for the boolean flag here?

Reply:
Done.